internal/api/config.go

```diff
@@ -1,5 +1,5 @@
 package api
 
 type Config struct {
-	Port string
```

Comment:
Could we include an `env:"PORT"` tag here too? I think we'll supply much of the config in prod via environment variables.

Reply:
Sure. Do we want to use a more specific name such as `SERVER_PORT` for the env variable?

Reply:
I prefer `SERVER_PORT`. We'll need to modify the [Terraform config](https://github.com/omshub/core-api/blob/main/ops/main.tf#L31) to add `SERVER_PORT` as an env var (DigitalOcean was automatically mapping `http_port` to `PORT` for us).